charts/brigade/templates/api-deployment.yaml

```diff
@@ -28,10 +28,12 @@ spec:
           httpGet:
             path: /healthz
             port: {{ .Values.api.service.internalPort }}
+{{ if .Values.api.linvenessProbe }}{{ toYaml .Values.api.livenessProbe | indent 10 }}{{ end }}
```

Comment:
Note that if you omit the if-end, you end up with the following error when `helm install` is run:

```
Error: UPGRADE FAILED: YAML parse error on brigade/templates/api-deployment.yaml: error converting YAML to JSON: yaml: line 31: could not find expected ':'
```

It is likely to be happening due to the empty `.livenessProbe` key producing `null` or alike, which breaks yaml.

Reply:
FYI: A similar implementation in the wild: https://github.com/Azure/helm-charts/blob/4f11be40cc7a47e776faca3fa566eb08c54cfb3b/drupal/templates/deployment.yaml#L81